docs/sources/tempo/operations/best-practices.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Best practices for traces
+menuTitle: Best practices
+description: Learn about the best practices for traces
+weight: 20
+---
+
+# Best practices for traces
+
+This page provides some general best practices for tracing.
+
+## Span and resource attributes
+
+[Traces]({{< relref "../traces" >}}) are built from spans, which denote units of work such as a call to, or from, an upstream service. Spans are constructed primarily of span and resource attributes. 
+Spans also have a hierarchy, where parent spans can have children or siblings. 
+
+In the screenshot below, the left side of the screen (1) shows the list of results for the query. The right side (2) lists each span that makes up the selected trace. 
+
+<p align="center"><img src="getting-started/assets/trace-explore-spans.png" alt="Trace example"></p>
+
+A **span attribute** is a key/value pair that provides context for its span. For example, if the span deals with calling another service via HTTP, an attribute could include the HTTP URL (maybe as the span attribute key `http.url`) and the HTTP status code returned (as the span attribute `http.status_code`). Span attributes can consist of varying, non-null types.
+
+Unlike a span attribute, a **resource attribute** is a key/value pair that describes the context of how the span was collected.
```

Comment:
Maybe add one sentence here to give more details.

... context of how the span was collected. Generally these attributes describe the process that created the span.

Reply:
```suggestion
Unlike a span attribute, a **resource attribute** is a key/value pair that describes the context of how the span was collected. Generally, these attributes describe the process that created the span.
```